test/modules/DirectMethodSender/DirectMethodLocalSender.cs

```diff
@@ -46,9 +46,9 @@ public sealed class DirectMethodLocalSender : DirectMethodSenderBase
                 logger);
         }
 
-        internal override async Task<int> InvokeDeviceMethodAsync(string deviceId, string targetModuleId, CancellationToken none)
+        internal override async Task<int> InvokeDeviceMethodAsync(string deviceId, string targetModuleId, string methodName, CancellationToken none)
```

Comment:
If you add methodName in this PR, you need to add it to DMReceiver as well.  I would suggest to not have this change in this PR as this PR is getting bigger.  BTW the current implementation of DMReceiver doesn't support connectivity test scenario; as it doesn't get sequence number from DM call.

Reply:
you're correct. The DMRecv is doing its own counting for number of direct method it receives. 
I'll get back to it once Dylan finalize what is needed to verify the result. 

The methodName, I think is independent of DMRecv.  The DMSender should be able to send whatever direct method it wants. (Though, if you want it to talk to DMRecv, it should only send the HelloWorld method).

Reply:
Always try to commit the work is ready to ship.  Currently I treat DM Receiver is broken in connectivity test scenario, please fix it in this PR.  Please agree and finalize the design / expected result structure with Dylan asap.

Reply:
Updated.